app/views/layouts/_header.html.erb

```diff
@@ -15,6 +15,8 @@
   		<ul class="nav navbar-nav">
   			<% if user_signed_in? %>
 
+          <li> <%= link_to "Пригласить", new_invitation_path if current_user.admin? %> </li>
```

Comment:
Should I place the link inside the Profile dropdown then? As for popups, it's a good idea to use them. By the way, it could be a great example for #45.